test/common/config/filesystem_subscription_test_harness.h

```diff
@@ -33,7 +33,11 @@ class FilesystemSubscriptionTestHarness : public SubscriptionTestHarness {
         api_(Api::createApiForTest(stats_store_)), dispatcher_(api_->allocateDispatcher()),
         subscription_(*dispatcher_, path_, stats_, *api_) {}
 
-  ~FilesystemSubscriptionTestHarness() { EXPECT_EQ(0, ::unlink(path_.c_str())); }
+  ~FilesystemSubscriptionTestHarness() {
+    if (::access(path_.c_str(), F_OK) != -1) {
```

Comment:
Why is this needed?

Reply:
I added a test, where deliverConfigUpdate is never called. So the config file is not even created, and unlink fails. All other tests call deliverConfigUpdate, so it was not a problem until now.